compliance_checker/cf/util.py

```diff
@@ -321,16 +325,27 @@ def create_cached_data_dir():
 
 def units_known(units):
     try:
+        # FIXME: pyudnits2 fails with dates.
```

Comment:
Yes, this is something that we should think about. Since it is in scope for `udunits2`, we should have a good answer in `pyudunits2`, even if that answer is to use `cftime`.

Ref https://github.com/pelson/pyudunits2/issues/4